wrappers/obj-c/CMakeLists.txt

```diff
@@ -29,20 +29,24 @@ endif()
 set (PLATFORM_LIBS "")
 # Add flags for obtaining system UUID via IOKit
 if (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
-set (PLATFORM_LIBS "-framework Foundation -framework CoreFoundation -framework IOKit -framework Network")
+set (PLATFORM_LIBS "-framework Foundation -framework CoreFoundation -framework IOKit -framework Network -framework SystemConfiguration")
```

Comment:
Why do you need to add SystemConfiguration?
Since this is a framework, this would require adopters to also add such framework

Reply:
This is being brought in due to third_party/Reachability code. Without this, the sample `main.mm` file fails with the linker errors below. Can you share what the recommendation here would be?
```
Consolidate compiler generated dependencies of target sample
[ 8%] Building CXX object CMakeFiles/sample.dir/main.mm.o
[ 16%] Building CXX object CMakeFiles/sample.dir/ODWLogManager.mm.o
[ 25%] Building CXX object CMakeFiles/sample.dir/ODWLogger.mm.o
[ 33%] Building CXX object CMakeFiles/sample.dir/ODWLogConfiguration.mm.o
[ 41%] Building CXX object CMakeFiles/sample.dir/ODWPrivacyGuardInitConfig.mm.o
[ 50%] Building CXX object CMakeFiles/sample.dir/ODWPrivacyGuard.mm.o
[ 58%] Linking CXX executable sample
Undefined symbols for architecture x86_64:
"_SCNetworkReachabilityCreateWithAddress", referenced from:
+[ODWReachability reachabilityWithAddress:] in ODWReachability.m.o
"_SCNetworkReachabilityCreateWithName", referenced from:
+[ODWReachability reachabilityWithHostname:] in ODWReachability.m.o
"_SCNetworkReachabilityGetFlags", referenced from:
-[ODWReachability isReachable] in ODWReachability.m.o
-[ODWReachability isReachableViaWiFi] in ODWReachability.m.o
-[ODWReachability connectionRequired] in ODWReachability.m.o
-[ODWReachability isConnectionOnDemand] in ODWReachability.m.o
-[ODWReachability isInterventionRequired] in ODWReachability.m.o
-[ODWReachability reachabilityFlags] in ODWReachability.m.o
"_SCNetworkReachabilitySetCallback", referenced from:
-[ODWReachability startNotifier] in ODWReachability.m.o
-[ODWReachability stopNotifier] in ODWReachability.m.o
"_SCNetworkReachabilitySetDispatchQueue", referenced from:
-[ODWReachability startNotifier] in ODWReachability.m.o
-[ODWReachability stopNotifier] in ODWReachability.m.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [sample] Error 1
make[1]: *** [CMakeFiles/sample.dir/all] Error 2
make: *** [all] Error 2
```